cmd/plume/specs.go

```diff
@@ -93,6 +107,16 @@ var (
 				Publish:     "coreos_production_gce.txt",
 				Limit:       25,
 			},
+			Azure: azureSpec{
+				Image:             "coreos_production_azure_image.vhd.bz2",
+				StorageAccount:    "coreos",
+				Containers:        []string{"publish", "pre-publish"},
+				Label:             "CoreOS alpha",
```

Comment:
this is simply mirroring https://github.com/coreos/scripts/blob/master/oem/azure/set-image-metadata.sh#L31, same for the others